### PR TITLE
Add demo data for op_profile view

### PR DESCRIPTION
**Pull request by sam-mccall**
The data is about the minimal set that exercises all current features (and #484)

For brevity, it omits some "required" stuff we don't currently use, like the byProgramStructure tree, and the raw{Time,Flops} metrics.

@ioeric

### CODE REVIEW

File: tensorboard/plugins/profile/profile_demo.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
 from tensorboard.plugins.profile import profile_demo_data
 from tensorboard.plugins.profile import profile_plugin
 from tensorboard.plugins.profile import trace_events_pb2
+from shutil import copyfile
```
Comment (chihuahua) [inline tensorboard/plugins/profile/profile_demo.py]: Could you replace with `import shutil` and move it near to `import os`? Native imports come first, and python style is to import the module, then use it.

File: tensorboard/plugins/profile/profile_demo.op_profile.json [hunk 2]
```diff
@@ -0,0 +1,95 @@
+{
```
Comment (chihuahua) [inline tensorboard/plugins/profile/profile_demo.op_profile.json]: Is this expected? I see nothing in the parentheses.

knxthhfajij

Comment (chihuahua) [reply tensorboard/plugins/profile/profile_demo.op_profile.json]: Also, "is overhead, and a good target for optimization" is a dangling half-sentence. I think something might be missing there.

**Comment by sam-mccall** [comment]
Good catch - I left too much out when constructing the data by hand, and the "_worstOp" calculation was crashing.
Added the missing empty 'children' arrays for the leaf nodes, it now looks like this:
image

File: tensorboard/plugins/profile/profile_demo.py (hunk 1 above)
Comment (chihuahua) [reply tensorboard/plugins/profile/profile_demo.py]: I just have this 1 comment, and then lets merge. Thanks!

Comment (sam-mccall) [reply tensorboard/plugins/profile/profile_demo.py]: Done. (Sorry, not sure how I missed this)

**Pull request merged**